### ISSUE TITLE
deb for debian/ubuntu

### ISSUE THREAD

**Issue by dmd** [rupa/z#96]
Someone should step up and be a package maintainer so z is part of Ubuntu/Debian. I don't really know how that stuff works, though. Politics etc.

**Comment by siddharthist** [comment]
Yeah, it's difficult. [Here's how it works](https://www.debian.org/doc/manuals/maint-guide/index.en.html).
